getting-started/scaling-out.md

```diff
@@ -0,0 +1,172 @@
+  TimescaleDB extension in it.
+  
+* Ensure that the local user has `USAGE` privilege on the `timescaledb_fdw` 
+  foreign data wrapper on the access node.
+  
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+SELECT add_data_node('node2', host => 'dn2.example.com',
+  password=>'<remote_password>', bootstrap_user=>'<superuser>',
+  bootstrap_password=>'<superuser_password>');
+```
+
+>:TIP: Any additional users that will access a distributed hypertable 
+currently need their own user mappings per data node with 
+a user and password option, which can be created with
+`CREATE USER MAPPING FOR` statement.
+
+Deleting a data node is done in a similar manner.
+
+```sql
+SELECT delete_data_node('node1');
+```
+>:TIP: Note that a data node cannot be deleted if it contains data for a
+hypertable.
+
+### Information Schema for Data Nodes
+
+The data nodes that have been added to the database can be found by
+querying the
+[`timescaledb_information.data_node`][timescaledb_information-data_node] view.
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertables with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a distributed hypertable, you need to have added at least one data node
+before starting.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time', 'location');
+```
+
+>:WARNING: If there are no data nodes available, you will get an error
+>when executing `create_distributed_hypertable` and the distributed
+>hypertable will not be created.
+
+You can now insert data into the distributed hypertable and
+it will automatically be partitioned on the available data nodes
+by using the provided space partition. You
+can find more information for how work with data in hypertables in the
+section [Creating Hypertables][creating-hypertables].
+
+### Changing the Number of Data Nodes for a Distributed Hypertable
+
+By default, distributed hypertables will use all available data nodes,
+but individual data nodes can be added and removed from already
+created distributed hypertables using
+[`attach_data_node`][attach_data_node] and
+[`detach_data_node`][detach_data_node].
+
+For example, if you add a new data node to the cluster, it will not automatically be
```

Comment:
Should we say "distrubuted database" instead of cluster?

Reply:
This document doesn't use distributed database, but uses cluster more. If it will be better to use distributed database across the document, then I suggest to do it as a separate PR.

Reply:
I think we should either use "multinode cluster" or "distributed database" to disambiguate this from a postgres cluster.

Reply:
Thank you for clarifying. This is an important point. I will check with other clustering docs.

Reply:
[Tutorial](https://github.com/timescale/docs.timescale.com-content/blob/clustering/tutorials/clustering.md) uses distributed database, so I switched to it too. It was actually not so many places.